lib/ui/text.dart

```diff
@@ -644,6 +656,58 @@ class ParagraphStyle {
   }
 }
 
+/// A single text shadow.
+/// 
+/// Multiple shadows are stacked together in a [TextStyle].
+class TextShadow {
+  static const int _kColorDefault = 0xFF000000;
+
+  const TextShadow({
+    this.color = const Color(0xFF000000),
+    this.offset = Offset.zero,
+    this.blurRadius = 0.0,
+  });
+
+  /// The color of the shadow.
+  final Color color;
```

Comment:
this doc violates our style guide (which requires that docs be more than what you can guess from the name).

See the style guide for writing prompts for docs.